src/cli/MeasureUpdateCommand.cpp

```diff
@@ -88,6 +97,203 @@ namespace cli {
                                               ->option_text("PORT")
                                               ->excludes(directoryPathOpt);
 
+    {
+      auto* newMeasureSubCommand = measureCommand->add_subcommand("new", "Create a new measure");
+
+      // TODO: this includes 'UtilityMeasure' which I don't think we want to include
+      std::vector<std::string> measureTypeNames;
+      {
+        const auto& m = openstudio::MeasureType::getNames();
+        std::transform(m.cbegin(), m.cend(), back_inserter(measureTypeNames), [](const auto& x) { return x.second; });
+        // fmt::print("measureTypeNames={}\n", measureTypeNames);
+        [[maybe_unused]] auto* measureTypeOpt =
+          newMeasureSubCommand
+            ->add_option("-t,--type", opt->newMeasureOpts.measureType,
+                         fmt::format("Type of Measure [Default: '{}']: {}", opt->newMeasureOpts.measureType.valueName(), measureTypeNames))
+            ->option_text("measureType")
+            ->check(CLI::IsMember(measureTypeNames));
```

Comment:
Throughout the options, I'm going to validate input.

eg try to pass `measure new --type Badval` and you'll get:

```
--type: Badval not in {ModelMeasure,EnergyPlusMeasure,UtilityMeasure,ReportingMeasure}
```